lib/_http_server.js

```diff
@@ -516,14 +516,23 @@ function connectionListener(socket) {
     }
 
     if (req.headers.expect !== undefined &&
-        (req.httpVersionMajor == 1 && req.httpVersionMinor == 1) &&
-        continueExpression.test(req.headers['expect'])) {
-      res._expect_continue = true;
-      if (EventEmitter.listenerCount(self, 'checkContinue') > 0) {
-        self.emit('checkContinue', req, res);
+        (req.httpVersionMajor == 1 && req.httpVersionMinor == 1)) {
+      if (continueExpression.test(req.headers.expect)) {
+        res._expect_continue = true;
+
+        if (EventEmitter.listenerCount(self, 'checkContinue') > 0) {
```

Comment:
@mscdex There is one other use of the `EventEmitter.listenerCount` in the file. Should I update it as well, or just change it in this one spot?

Not sure how tightly scoped y'all like to keep the PRs.